docs/UserGuide.md

```diff
@@ -3,213 +3,268 @@ layout: page
+  * You can remove all the tags of a food item by typing `t/` without specifying any tags after it.
 
-Examples:
-* `edit 1 d/baked beans e/1-1-2020` Edits the food description and expiry date of the 1st food item to be `baked beans` and `1-1-2020` respectively.
-* `edit 2 d/canned tuna q/0.5 can t/` Edits the food description of the 2nd food item to be `canned tuna`, quantity to `0.5 can` and clears all existing tags.
+**Examples:**
+* `edit 1 d/baked beans e/1-1-2020` edits the food description and expiry date of the 1st food item to be `baked beans` and `1-1-2020` respectively.
+* `edit 2 d/canned tuna q/0.5 can t/` edits the food description of the 2nd food item to be `canned tuna`, quantity to `0.5 can` and clears all existing tags.
 
-### [Changing the quantity of a food item](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#features) : `changeqty`
+<div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html">^ Back to top</a></div>
 
-Changes the quantity of an existing food item in the food inventory.
+## Changing the quantity of a food item
 
-Format: `changeqty INDEX a/AMOUNT`
+The `changeqty` command changes the quantity of an existing food item in your food inventory.
+Use this command if you have bought new food items or used/discarded some existing food items.
 
-* The index refers to the number shown in the displayed food list. It must be a positive unsigned integer i.e. 1, 2, 3 and so on.
+**Format:** `changeqty INDEX a/AMOUNT`
+
+* The index refers to the index number shown in the displayed food list.
+  * The index **must be a positive integer** 1, 2, 3, …
 * The amount is the quantity of a food item you want to change by. It is compulsory and can be any non-zero signed number.
-* Choose an amount such that the final quantity should not be less than or equal to zero.
-* The amount can be specified up to a maximum of 2 decimal places.
-* Do not specify the unit of the food item. The existing unit is used instead.
+  * Choose an amount such that the final quantity is not less than or equal to zero.
+  * The amount can be specified up to a maximum of 2 decimal places.
+* Do not specify the unit of the food item. The existing unit will be used instead.
 
-Examples:
+**Examples:**
 * `changeqty 1 a/+1` increases the quantity of the 1st food item by 1.
 * `changeqty 2 a/-2` decreases the quantity of the 2nd food item by 2.
 
-### [Undoing previous command](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#features) : `undo`
-Restores the food inventory to a state before an undoable command was executed.
+<div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html">^ Back to top</a></div>
+
+## Undoing previous command
 
-* Undoable commands: commands that modify the food inventory's content (`add`, `delete`, `edit` and `clear`)
+The `undo` command restores the food inventory to a state before an undoable command was executed.
```

Comment:
Very minor, but perhaps this can be "your food inventory"? For consistency.

Reply:
Edited. Thanks.